src/server/biosecurity-map/upload-progress/index.test.js

```diff
@@ -34,18 +34,22 @@ describe('#UploadPlan', () => {
 
   beforeEach(async () => {
     session = await SessionTestHelper.create(server)
-    await session.setState(uploadConfig.questionKey, {
-      statusUrl: 'http://localhost/status'
+    await session.setState('application', {
```

Comment:
I don't honestly know how this ever worked, looking at the code.  I think this `await` statement might be dead code.